.github/workflows/ci.yml

```diff
@@ -17,6 +17,12 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+      - uses: dtolnay/rust-toolchain@stable
+        with:
+          components: clippy
+      - name: Linting
+        run: cargo clippy -- -D warnings
```

Comment:
Please add `-- -D warnings` to L43~L47

Reply:
I added some deny.toml files, modified Clippy.